src/main/java/com/scriptbasic/utility/functions/MathFunctions.java

```diff
@@ -216,9 +220,30 @@ public static double rint(final double a) {
         return 0.0;
     }
 
-    @BasicFunction(substituteClass = java.lang.Math.class, classification = com.scriptbasic.classification.Math.class)
-    static public double round(final double x) {
-        return 0.0;
+    /**
+     * Returns a number rounded to a specified number of decimal places.
+     * 
+     * This function returns something commonly referred to as bankers rounding. 
+     * So be careful before using this function. Same behavior has 'round' in VBA.
+     * 
+     * @param value Numeric value being rounded
+     * @param numdecimalplaces Number indicating how many places to the right of the decimal 
+     *        are included in the rounding.
+     * @return rounded value
+     */
+    @BasicFunction(classification = com.scriptbasic.classification.Math.class)
+    static public Object round(final double value, Integer numdecimalplaces) {
+        
+        BigDecimal bd = BigDecimal.valueOf(value);
+        
+        if(numdecimalplaces==null||numdecimalplaces==0) {
```

Comment:
How about using `Math.round(double)` in case there is no second parameter? Just to keep backward compatibility in the implementation. What is your opinion about that? Am I too cautious?

Reply:
I've extracted rounding algorithm to the static variable. Default algorithm can be changed.

Math.round(double) does not allow to specify rounding algorithm so I would prefer to use rounding with BigDecimal.